### PR TITLE
Document SemigroupsTestExtreme

### PR DESCRIPTION
**Pull request by mtorpey**
As requested in #298.

### CODE REVIEW

File: doc/utils.xml [hunk 1]
```diff
@@ -71,6 +71,26 @@
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="SemigroupsTestExtreme">
+  <ManSection>
+    <Func Name = "SemigroupsTestExtreme" Arg = ""/>
+    <Returns>A list indicating which tests passed and failed and the time take
+      to run each file.</Returns>
+    <Description>
+      This function should be called with no argument to run some big tests,
```
Comment (james-d-mitchell) [inline doc/utils.xml]: "big" -> "long running"

File: doc/utils.xml [hunk 2]
```diff
@@ -71,6 +71,26 @@
...
+      which could be used to benchmark &Semigroups; or test your hardware. These
+      tests should take no more than around half an hour to complete.  To
+      quickly test that &Semigroups; is installed correctly use
+      <Ref Func = "SemigroupsTestInstall"/>, or to test all aspects of the
+      package use <Ref Func = "SemigroupsTestStandard"/>.<P/>
+
+      Each test file is run twice, once when the methods for
+      <Ref Filt="IsActingSemigroup"/> are enabled and once when they are
```
Comment (james-d-mitchell) [inline doc/utils.xml]: "methods for semigroups satisfying IsActingSemigroup"

File: doc/z-chap02.xml [hunk 3]
```diff
@@ -163,12 +163,12 @@ make]]></Listing>
...
     In this section we describe how to test that &Semigroups; is working as
-    intended.  To test that &Semigroups; is installed correctly use
-    <Ref Func = "SemigroupsTestInstall"/> or for more extensive tests use
-    <Ref Func = "SemigroupsTestStandard"/>. Please note that it will take a few
-    seconds for <Ref Func = "SemigroupsTestInstall"/> to finish and it may take
-    several minutes for <Ref Func = "SemigroupsTestStandard"/> to finish.
-    <P/>
+    intended.  To quickly test that &Semigroups; is installed correctly use
+    <Ref Func = "SemigroupsTestInstall"/> (this will take a few seconds).
+    For more extensive tests use <Ref Func = "SemigroupsTestStandard"/> (this
```
Comment (james-d-mitchell) [inline doc/z-chap02.xml]: Remove parentheses.

**Comment by mtorpey** [comment]
Changed as suggested.

**Pull request merged**